src/transaction/SimulationResults.scss

```diff
@@ -120,4 +120,13 @@
       background-color: rgb(12, 100, 12);
     }
   }
+
+  // Source: https://stackoverflow.com/questions/11679567/using-css-for-a-fade-in-effect-on-page-load
+  .fadeIn {
+    -webkit-animation: fadein .5s; /* Safari, Chrome and Opera > 12.1 */
+     -moz-animation: fadein .5s; /* Firefox < 16 */
+      -ms-animation: fadein .5s; /* Internet Explorer */
+       -o-animation: fadein .5s; /* Opera < 12.1 */
+          animation: fadein .5s;
+  }
```

Comment:
Does not seem to be used anywhere. If you want to keep it, rewrite it yourself (I'm unsure about licensing even snippets from SO) and the browser prefix properties are unnecessary (only `animation` is needed).